job-dsls/jobs/prod/prod_offline_repo_builder.groovy

```diff
@@ -18,7 +18,7 @@ job("${folderPath}/offline-repo-builder") {
 
     parameters {
         stringParam("MANIFEST_URL", "", "Offliner manifest URL")
-        stringParam("RELEASE_REPO_GROUP_URL", "http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/content/groups/offline-repo-group-7-11/", "Scratch repositories group for the release")
+        stringParam("RELEASE_REPO_GROUP_URL", "\${env.BXMS_QE_NEXUS}/content/groups/offline-repo-group-7-11/", "Scratch repositories group for the release")
```

Comment:
env variables can't be used here, please move this value inside the job itself

Reply:
according to https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/custom/job/emingora/job/envtest/2/console it can be used, so I agree on leave it as it is.